api/models.py

```diff
@@ -128,14 +128,14 @@ class Event(BaseInfo):
 
     title = models.CharField(max_length=100)
     description = models.TextField()
-    venue = models.TextField(default='Pekas Night Club')
+    venue = models.TextField()
     date = models.CharField(default='September 10, 2017', max_length=200)
```

Comment:
Why the default date and time?

Reply:
I remember we had an issue with the way the date was coming from the backend then. I left this as a precautionary measure incase we didn't fix it. @andela-iukwuoma